tests/test_cli.py

```diff
@@ -0,0 +1,146 @@
+import io
```

Comment:
E2E testing is good, but making network requests is too slow. It is enough to monkeypatch `JavaServer.status`, `.query`, `.ping`, etc.

We also have plans on integration testing, but unfortunately, we need some kind of server to host test servers.